storage-resize-images/extension.yaml

```diff
@@ -243,3 +243,14 @@ params:
         value: false
     default: false
     required: true
+
+
+  - param: OUTPUT_OPTIONS
+    label: Output options for selected formats
+    description: >
+      Provide a optional output option stringified object containing Sharp Output Options for selected image types conversion.
+      eg. `{"jpeg": { "quality": 5, "chromaSubsampling": "4:4:4" }, "png": { "pallete": true }}`
+    type: string
+    validationRegex: ^({(.*?)})$
+    validationErrorMessage: Please provide a valid json object. Expected to start with `{`.
```

Comment:
`expected to start with` presumes this is only invalid way for writing JSON according to this validation.

Are there other cases for invalid regex and if so can we remove that sentence from the `validationErrorMessage`?